src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -12,7 +12,7 @@ <h2>{{ addon_name }}</h2>
           </a>
         {% endwith %}
       {% else %}
-        <img class="screenshot thumbnail" src="{{ addon.thumbnail_url }}"
+        <img class="screenshot thumbnail" src="{{ addon.get_icon_url(150) }}"
```

Comment:
this is just a drive-by fix because the error showed up in the console each time I re-loaded the page